modules/persistent-storage-csi-gcp-pd-encrypted-pv.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+[id="persistent-storage-csi-gcp-pd-encrypted-pv_{context}"]
+= Creating a custom-encrypted persistent volume
+
+When you create a `PersistentVolume` object, {product-title} creates a persistent volume (PV). You can add a custom encryption key in Google Cloud Platform (GCP) to protect PVs in your cluster by encrypting a newly created PV.
+
+You can create an attached volume that uses customer-managed encryption keys (CMEK) on a new or existing cluster by using a new or existing Cloud KMS key.
+
+.Prerequisites
+* You are logged in to a running {product-title} cluster.
+* You have created a PVC using a the GCP PD CSI driver.
+* You have created a GCP Cloud KMS key ring and key version.
+
+For more information about CMEK and Cloud KMS resources, see link:https://cloud.google.com/kubernetes-engine/docs/how-to/using-cmek[Using customer-managed encryption keys (CMEK)].
+
+.Procedure
+To create a custom-encrypted PV, complete the following steps:
+
+. Create a storage class with the Cloud KMS key. The following example enables dynamic provisioning of encrypted volumes:
++
+[source,yaml]
+--
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: csi-gce-pd-cmek
+provisioner: pd.csi.storage.gke.io
+volumeBindingMode: "WaitForFirstConsumer"
+allowVolumeExpansion: true
+parameters:
+  type: pd-standard
+  disk-encryption-kms-key: projects/key-project-id/locations/location/keyRings/key-ring/cryptoKeys/key <1>
+--
+<1> This field must be the resource identifier for the key that will be used to encrypt new disks. Values are case-sensitive.
++
+[NOTE]
+====
+You cannot add the `disk-encryption-kms-key` parameter to an existing storage class. However, you can delete the storage class and recreate it with the same name and a different set of parameters. If you do this, the provisioner of the existing class must be `pd.csi.storage.gke.io`.
+====
+
+. Deploy the storage class on your GKE cluster using the `oc` command:
```

Comment:
Beware, `GKE` is Google Kubernetes Engine, Google's product for *container* orchestration and direct competitor of OCP. We don't support that and we should not refer to it either.

We support GCE (Google Compute Engine), which is *virtual machine* orchestration, where customers deploy OCP.

Both GCE and GKE are part of GCP (Google Cloud Platform), overall umbrella over all Google's cloud services. Hence the driver is called "GCP PD CSI driver" (PD = Persistent Disk).

Reply:
Thanks, @jsafrane. I missed that update here and have changed to OCP. Thanks for the helpful arch explanation.

I've also updated the `Provisioner` parameter in the samples (`s/pd.csi.storage.gke.io/pd.csi.storage.k8s.io`). Please confirm whether or not that is the correct label.